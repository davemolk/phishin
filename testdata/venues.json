{"success":true,"total_entries":666,"total_pages":34,"page":1,"data":[{"id":68,"slug":"the-base-lodge-johnson-state-college","name":"The Base Lodge, Johnson State College","other_names":[],"latitude":44.558803,"longitude":-72.577842,"location":"Johnson, VT","city":"Johnson","state":"VT","country":"USA","shows_count":2,"show_dates":["1988-03-11","1989-04-14"],"show_ids":[374,537],"updated_at":"2023-09-25T22:02:15Z"},{"id":11,"slug":"the-academy","name":"The Academy","other_names":[],"latitude":40.783515,"longitude":-73.958766,"location":"New York, NY","city":"New York","state":"NY","country":"USA","shows_count":1,"show_dates":["1991-07-15"],"show_ids":[472],"updated_at":"2013-03-24T03:17:31Z"}]}