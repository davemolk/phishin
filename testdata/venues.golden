Venue:                                 Location:     Show Count:
The Base Lodge, Johnson State College  Johnson, VT   2
The Academy                            New York, NY  1

Total Entries: 666  Total Pages: 34  Result Page: 1
